test/functional/users_controller_test.rb

```diff
@@ -322,6 +322,7 @@ class UsersControllerTest < ActionController::TestCase
   end
 
   test "#login renders login page" do
+    session.clear
```

Comment:
Sure @domcleal I am sorry I didn't quite get you. Could you please give an example?